ChangeLog.d/remove-pk-rsa-alt.txt

```diff
@@ -0,0 +1,2 @@
+API changes
+   * Remove MBEDTLS_PK_RSA_ALT from the system.
```

Comment:
IMO `the system` in this sentence is a bit generic. Either you entirely skip the `from the system` part, or you write something like `Remove MBEDTLS_PK_RSA_ALT from PK module`.